src/integrationTest/resources/tests/arm/properties/all_properties/annotation-record.properties

```diff
@@ -0,0 +1,20 @@
+bf_001=OBJECT_TYPE
+bf_002=CASE_NUMBERS
+bf_003=FILE_TYPE
+bf_004=HEARING_DATE
+bf_005=CHECKSUM
+bf_006=TRANSCRIPT_REQUEST
+bf_007=TRANSCRIPT_TYPE
+bf_008=TRANSCRIPT_URGENCY
+bf_009=COMMENTS
+bf_010=CREATED_DATE_TIME
+bf_011=CREATED_DATE_TIME
+bf_012=OBJECT_ID
+bf_013=PARENT_ID
+bf_014=CHANNEL
+bf_015=MAX_CHANNELS
+bf_016=UPLOADED_BY
+bf_017=START_DATE_TIME
```

Comment:
Should this be `END_DATE_TIME`?
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1725104200&preview=/1725104200/1753715590/image001.png#ARMLLD-Metadata

Reply:
As with previous message annotation documents don't have start and ends.

Reply:
Again this is the all properties so its just testing that values are set only where there are values. All Properties wouldn't be used but its just to make sure where there is no mapping, no value gets set